webv2/web.go

```diff
@@ -1953,15 +1850,15 @@ func checkPrimaryKeyPrefix(table string, refTable string, fk ddl.Foreignkey, tab
 	canInterleavedOnRename := []string{}
 	for i := 0; i < len(diff); i++ {
 
-		parentColIndex := utilities.IsColumnPresent(fk.ReferColumns, diff[i].Col)
+		parentColIndex := utilities.IsColumnPresent(fk.ReferColumnIds, diff[i].ColId)
 		if parentColIndex == -1 {
 			continue
 		}
-		childColIndex := utilities.IsColumnPresent(childPkCols, fk.Columns[parentColIndex])
+		childColIndex := utilities.IsColumnPresent(childPkCols, fk.ColIds[parentColIndex])
 		if childColIndex == -1 {
-			canInterleavedOnAdd = append(canInterleavedOnAdd, fk.Columns[parentColIndex])
+			canInterleavedOnAdd = append(canInterleavedOnAdd, fk.ColIds[parentColIndex])
 		} else {
-			canInterleavedOnRename = append(canInterleavedOnRename, fk.Columns[parentColIndex])
+			canInterleavedOnRename = append(canInterleavedOnRename, fk.ColIds[parentColIndex])
```

Comment:
Not seeing interleave suggestions for add, rename or change PK order in the UI

Reply:
Updated. Please review them.